src/state/reducers/scenario/ScenarioReducer.js

```diff
@@ -20,11 +20,11 @@ export const scenarioListReducer = createReducer(scenarioListInitialState, (buil
       state.data = action.list;
       state.status = STATUSES.SUCCESS;
     })
-    .addCase(SCENARIO_ACTIONS_KEY.UPDATE_SCENARIO_STATE, (state, action) => {
+    .addCase(SCENARIO_ACTIONS_KEY.UPDATE_SCENARIO, (state, action) => {
       state.data = state.data.map(scenarioData => {
-        // Replace state in data for the scenario to update
+        // Replace state and lastun in data for the scenario to update
         if (scenarioData.id === action.data.scenarioId) {
-          return { ...scenarioData, state: action.data.scenarioState };
+          return { ...scenarioData, state: action.data.scenarioState, lastRun: action.data.lastrun };
```

Comment:
Shouldn"t it be lastRun instead of lastrun ?

Reply:
Fixed all occurences of scenarioRun case

Reply:
Fixed